.github/workflows/release.yml

```diff
@@ -22,7 +22,7 @@ jobs:
       - name: Install pnpm
         uses: pnpm/action-setup@v3
         with:
-          version: 8
+          version: 9.23
```

Comment:
Is this version meant to not matching compressed-size & main? Is there a reason we're not just using `9`?